tests/python_package_test/test_engine.py

```diff
@@ -966,6 +966,26 @@ def test_continue_train_multiclass():
     assert evals_result['valid_0']['multi_logloss'][-1] == pytest.approx(ret)
 
 
+def test_continue_train_different_feature_size():
```

Comment:
Please don't leave tests without any `assert_*` statements.
Refer to https://github.com/microsoft/LightGBM/pull/1973#issuecomment-458591675.
For example, here you can check for values of [`get_feature_name()`](https://lightgbm.readthedocs.io/en/latest/pythonapi/lightgbm.Dataset.html#lightgbm.Dataset.get_feature_name) and [`num_feature()`](https://lightgbm.readthedocs.io/en/latest/pythonapi/lightgbm.Booster.html#lightgbm.Booster.num_feature) methods.

Reply:
Resolved via [750d35c](https://github.com/microsoft/LightGBM/pull/5157/commits/750d35c62a7a86d36794685239cd19332a387c4a)